scio-test/src/main/scala/com/spotify/scio/testing/SCollectionMatchers.scala

```diff
@@ -68,89 +97,199 @@ trait SCollectionMatchers {
   private def serDeCycle[T: ClassTag](scollection: SCollection[T]): SCollection[T] = {
     val coder = scollection.internal.getCoder
     scollection
-      .map(e => CoderUtils.decodeFromByteArray(coder, CoderUtils.encodeToByteArray(coder, e)))
+      .map(
+        e =>
+          CoderUtils
+            .decodeFromByteArray(coder, CoderUtils.encodeToByteArray(coder, e)))
   }
 
+  /** SCollection assertion only applied to the specified window,
```

Comment:
Format multi-line docs like this to be consistent with the rest of the code base?
```scala
/**
 * Blah blah.
 */
```